envs/remove_relatives.yaml

```diff
@@ -0,0 +1,5 @@
+name: remove_relatives

```

Comment:
I cannot find usages of this conda environment. Should it be deleted?

Reply:
It's an env for remove_relatives workflow